README.md

```diff
@@ -40,4 +40,4 @@ If you need any help, feel free to [create an issue](https://github.com/flamesho
 
 ## Traditional Translation Way
 
-[Click here](https://github.com/flameshot-org/translation-instruction/blob/master/Traditional_Translation_Way.md).
\ No newline at end of file
+[Click here](https://github.com/flameshot-org/translation-instruction/blob/master/Traditional_Translation_Way.md).
```

Comment:
What has changes here?!

Reply:
can is be a bug in Github? I don't see any difference between those two lines.

Reply:
?! I didn't touch that line..... dang🤔

Reply:
Let's merge it then as nothing has changed as far as I can see.

Reply:
I guess... nothin' has changed 👀
Thank you.